CHANGELOG.md

```diff
@@ -55,6 +55,8 @@
 
 ### Snowpark Python API Updates
 
+- Updated `Session` class to be thread-safe. This allows concurrent dataframe transformations, dataframe actions, UDF and store procedure registration, and concurrent file uploads.
```

Comment:
this message is not clear about the change here. Please update the change log to reflect the actual change in this pr, and the reason to add this to help making the session class to be safe